auth/realm/base/pom.xml

```diff
@@ -85,6 +85,11 @@
             <groupId>org.wildfly.common</groupId>
             <artifactId>wildfly-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.santuario</groupId>
+            <artifactId>xmlsec</artifactId>
```

Comment:
We should not forget to make sure this is on classpath during execution and we know where it gets loaded from

Reply:
@Ashpan I know you were going to check this in Core. Has that been done?